go/log_messages.md

```diff
@@ -76,16 +86,19 @@ The fact gathering statement expresses what happens and in case the action fails
 field is added to get some tracing information from the carried error.
 
 ```go
-s.logger.Log("level", "debug", "message", "collecting metrics")
-
-for _, c := range s.collectors {
-	err := c.Collect(ch)
-	if err != nil {
-		s.logger.LogCtx(ctx, "level", "error", "message", "failed to collect metrics", "stack", fmt.Sprintf("%#v", microerror.Mask(err)))
+{
+	actionLogger := r.loggger.With("action", "collecting metrics")
+	s.logger.LogCtx(ctx, "level", "debug", "message", "start")
```

Comment:
I think you should use actionLogger instead of s.logger, in orde to output the "action" key on start and end messages.